### ISSUE TITLE
Missing label for confidence icon [Accessibility ⭐ ]

### ISSUE THREAD

**Issue by sumairq** [mozilla/perfcompare#414]
Labels for confidence icons are missing and need to be created so that `unknown confidence` `high confidence` `low confidence` states are properly read out by screen readers.
